curriculum/challenges/english/25-front-end-development/workshop-cat-photo-app/66984218720e29393b1adad2.md

```diff
@@ -47,7 +47,7 @@ const childrenOfSection = [...document.querySelector('section').childNodes];
 const foundElements = childrenOfSection.filter((child) => {
   return ['H2', 'A', 'P'].includes(child.nodeName);
 });
-assert.equal(foundElements.length, 3);
+assert.lengthOf(foundElements.length, 3);
```

Comment:
```suggestion
assert.lengthOf(foundElements, 3)
```
I'm not leaving anymore suggestions, if you use `assert.legnthOf`, don't use `.length`. Please fix the others